modules/ROOT/pages/delete-message-flows.adoc

```diff
@@ -0,0 +1,23 @@
+== Delete Message Flows
+
+You can delete message flows if they no longer serves their purpose, for example, if your relationship with your partner changes. You can delete message flows from a sandbox or production environment.
+
+Deleting a message automatically undeploys the flow. If you Delete Message Flows before it is deployed, the Mule apps are not affected. [*Vijay. What Mule apps is this referring to?*]
```

Comment:
```suggestion
Deleting a message from a flow automatically undeploys the flow, unless that flow has not yet been deployed. [*Vijay. What Mule apps is this referring to?*]
```
Do we need to document this here and in this way? The topic is deleting message flows; we do not want to suggest deleting a message to do so, do we? That sounds like an unintentional action and result.